README.md

```diff
@@ -9,3 +9,27 @@ git remote add upstream git@github.com:bitfinexcom/PARENT.git
 git remote -v
 
 git push origin master
+
+## Config validation
+
+If config file has corresponding `.example` file, by default `loadConf` will run a check for missing keys.
+
+If config is missing some keys that are present in `.example` file, program will exit and report all missing keys.
+
+`loadConf` accepts 3rd, optional validation object parameter:
+
+```js
+this.loadConf("cosmos.coin", "coin", {
+  symbol: {
+    required: true,
+    sameAsExample: true,
```

Comment:
What would be a good example for enabling `sameAsExample`?
Wouldn't it make sense not to include it in the config since it's a constant?

Reply:
I did what I was asked for 🤷‍♂️